pyanaconda/payload/dnfpayload.py

```diff
@@ -1125,3 +1153,74 @@ def postInstall(self):
 
     def writeStorageLate(self):
         pass
+
+class RepoMDMetaHash(object):
+    """Class that holds hash of a repomd.xml file content from a repository.
+    This class can test availability of this repository by comparing hashes.
+    """
+    def __init__(self, dnf_payload, repo):
+        self._repoId = repo.id
+        self._method = dnf_payload.data.method.method
+        self._urls = repo.baseurl
+        self._repomd_hash = ""
+
+    @property
+    def repoMDHash(self):
```

Comment:
I think we might want to use underscores in the method names of new code F26/master branch.